extensions/EXT/EGL_EXT_surface_compression.txt

```diff
@@ -0,0 +1,289 @@
+Name
+
+    EXT_surface_compression
+
+Name Strings
+
+    EGL_EXT_surface_compression
+
+Contributors
+
+    Jan-Harald Fredriksen, Arm
+    Lisa Wu, Arm
+    George Liu, Arm
+    Laurie Hedge, Imagination Technologies
+
+Contact
+
+    Jan-Harald Fredriksen (jan-harald.fredriksen 'at' arm.com)
+
+IP Status
+
+    No known IP claims.
+
+Status
+
+    Complete
+
+Version
+
+    Version 1 - November 15, 2021
+
+Number
+
+    EGL Extension #147
+
+Dependencies
+
+    These extensions are written against the wording of the EGL 1.5
+    specification (August 27, 2014).
+
+    This extension interacts with EGL_EXT_yuv_surface.
+
+Overview
+
+    Applications may wish to take advantage of framebuffer compression. Some
+    platforms may support framebuffer compression at fixed bitrates. Such
+    compression algorithms generally produce results that are visually lossless,
+    but the results are typically not bit exact when compared to a non-compressed
+    result.
+
+    This extension enables applications to opt-in to fixed-rate compression
+    for EGL window surfaces.
+
+    Compression may not be supported for all framebuffer formats. It can still
+    be requested for all formats and applications can query what level of compression
+    was actually enabled. 
+
+New Procedures and Functions
+
+    EGLBoolean eglQuerySupportedCompressionRatesEXT(
+           EGLDisplay dpy, EGLConfig config, const EGLAttrib *attrib_list,
+           EGLint *rates, EGLint rate_size, EGLint *num_rates);
+
+New Tokens
+
+    New attributes accepted by the <attrib_list> argument of
+    eglCreatePlatformWindowSurface and eglCreateWindowSurface:
+        EGL_SURFACE_COMPRESSION_EXT                     0x34B0

```

Comment:
The extension itself looks fine, but you will also need to add it to egl.xml before we merge this, including the new entry points and enums as well as the new \<extension> block; and also to registry.tcl so the index gets regenerated properly.